controller/image.yaml

```diff
@@ -0,0 +1,44 @@
+schema_version: 1
+
+name: "rhba-7/rhba70-controller"
+description: "Red Hat Business Automation Standalone Controller 7.0 container image"
+version: "7.0.0"
+from: "jboss-eap-7/eap71:latest"
+labels:
+    - name: "org.jboss.product"
+      value: "rhba-controller"
+    - name: "org.jboss.product.version"
+      value: "7.0.0"
+    - name: "org.jboss.product.rhba-controller.version"
+      value: "7.0.0"
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "rhba-controller"
+    - name: "JBOSS_RHBA_CONTROLLER_VERSION"
+      value: "7.0.0"
+    - name: "PRODUCT_VERSION"
+      value: "7.0.0"
+    - name: "ADD_ONS_DISTRIBUTION_ZIP"
+      value: "rhbas-7.0.0.ER1-add-ons.zip"
```

Comment:
shouldn't be value rhba-7.0.0... (instead of rhbas)?

Reply:
This build (rhbas er1) was the most recent one I was told about that is still stable on EAP 7.1. I need a new build before I can update this.